spec/requests/admin/users_spec.rb

```diff
@@ -8,27 +8,64 @@
   describe 'GET /admin/users' do
     subject { get admin_users_path }
 
-    context 'ユーザが管理者の場合' do
-      before { log_in_as(admin_user) }
+    context 'ログインしている場合' do
+      context 'ユーザが管理者の場合' do
+        context 'クエリパラメータにpageがない場合' do
+          xit 'ユーザー取得APIにoffset=0パラメータをつけて呼び出すこと' do
+            # TODO: specの内容を作成する
+          end
 
-      it '200が返って、アクセスできる' do
-        subject
-        expect(response).to be_successful
+          xit 'ステータスコード200とともに、先頭から10件分のユーザーを返すこと' do
+            # TODO: specの内容を作成する
+          end
+        end
+
+        context 'クエリパラメータにpageがある場合' do
+          context 'pageの値が0の場合' do
+            xit 'ユーザー取得APIにoffset=0パラメータをつけて呼び出すこと' do
+              # TODO: specの内容を作成する
+            end
+
+            xit 'ステータスコード200とともに、先頭から10件分のユーザーを返すこと' do
+              # TODO: specの内容を作成する
+            end
+          end
+
+          context 'pageの値が1の場合' do
+            xit 'ユーザー取得APIにoffset=0パラメータをつけて呼び出すこと' do
+              # TODO: specの内容を作成する
+            end
+
+            xit 'ステータスコード200とともに、先頭から10件分のユーザーを返すこと' do
+              # TODO: specの内容を作成する
+            end
+          end
+
+          context 'pageの値が2の場合' do
+            xit 'ユーザー取得APIにoffset=10パラメータをつけて呼び出すこと' do
+              # TODO: specの内容を作成する
+            end
+
+            xit 'ステータスコード200とともに、offsetされた10件分を除いて先頭から10件分のユーザーを返すこと' do
```

Comment:
ここ、先頭じゃないのに「先頭から10件分」って言葉を使っているので変な表現になっているのでは?
```suggestion
            xit 'ステータスコード200とともに、先頭から11件目〜20件目のユーザーを返すこと' do
```